lib/Controller/SignalingController.php

```diff
@@ -275,6 +275,7 @@ protected function getInputStream() {
 	 * https://github.com/nextcloud/spreed/wiki/Spreed-Signaling-API
 	 *
 	 * @PublicPage
+	 * @NoCSRFRequired
```

Comment:
Since this is OCS, it should not be necessary. If your requests fail, try adding a `OCS-APIRequest: true` header

Reply:
Thanks, didn't know about that header. If I set it, the `@NoCSRFRequired` can be removed - will remove that commit from the PR.